artifacts/mods/gl_highlighting.ssl

```diff
@@ -1,42 +1,53 @@
 /**
-   
-   Item Highlight mod. 
-   
+
+   Item Highlight mod.
+
    Previously was part of sfall itself, now a separate mod.
    Features:
-   - highlighting items, containers (optional) and dead bodies on the ground
+   - highlighting items, containers (optional) and dead bodies (optional) on the ground
    - configurable hotkey is used to trigger highlight
-   - only objects in direct line-of-sight of player are highlighted
+   - only objects in direct line-of-sight of player are highlighted (optional)
    - motion scanner is required to enable highlight (optional)
    - motion scanner charges are decreased on each use (optional)
-   
+
    NOTE: this script requires compiler from sfall modderspack with -s option
    (short circuit evaluation)
-   
+
    version 1.0
-   
+
 **/
 
 #include "main.h"
 //#include "..\..\..\!SRC\headers\define.h"
 
-#define CRITTER_IS_DEAD   (1)
+#define CRITTER_IS_DEAD    (1)
 #define PID_MOTION_SENSOR  (59)
 #define NO_HIGHLIGHT(obj)  (get_flags(obj) bwand FLAG_NOHIGHLIGHT)
 
 variable configSection := "Highlighting";
 variable highlightKey;
 variable alsoContainer;
+variable alsoCorpse;
+variable checkLOS;
 variable outlineColor;
 variable motionScanner;
 variable highlightFailMsg1;
 variable highlightFailMsg2;
 
 procedure ToggleHighlightObject(variable obj, variable enable) begin
-   if obj and (not enable or not obj_blocking_line(dude_obj, tile_num(obj), BLOCKING_TYPE_SHOOT)) then begin
-      if (alsoContainer and obj_item_subtype(obj) == item_type_container) or (not NO_HIGHLIGHT(obj)) then begin
-         if (enable) then set_outline(obj, outlineColor);
-         else set_outline(obj, 0);
+   if (checkLOS) then begin
+      if obj and (not enable or not obj_blocking_line(dude_obj, tile_num(obj), BLOCKING_TYPE_SHOOT)) then begin
+         if (alsoContainer and obj_item_subtype(obj) == item_type_container) or (not NO_HIGHLIGHT(obj)) then begin
+            if (enable) then set_outline(obj, outlineColor);
+            else set_outline(obj, 0);
+         end
+      end
+   end else begin
+      if obj then begin
+         if (alsoContainer and obj_item_subtype(obj) == item_type_container) or (not NO_HIGHLIGHT(obj)) then begin
+            if (enable) then set_outline(obj, outlineColor);
+            else set_outline(obj, 0);
```

Comment:
Heh, first thing I did was change to bright yellow as well. Simply because I was used to it and it looks "Fallout" to me, while the not so bright yellow is somehow... not really sticking out, dunno how to describe it.